Revisions for Manuscript (#24)
greenelab/text_mined_hetnet_manuscript: content/01.abstract.md

Unfortunately writing useful label functions requires substantial error analysis and is a nontrivial task: in early efforts to use data programming we found that producing each label function could take a few days.
Producing a biomedical knowledge base with multiple node and edge types could take hundreds or possibly thousands of label functions.
In this paper we sought to evaluate the extent to which label functions could be re-used across edge types. 
We used a subset of Hetionet v1 that centered on disease, compound, and gene nodes to evaluate this approach.
We compared a baseline distant supervision model with the same distant supervision resources added to edge-type-specific label functions, edge-type-mismatch label functions, and all label functions.
We confirmed that adding additional edge-type-specific label functions improves performance.
We also found that adding one or a few edge-type-mismatch label functions nearly always improved performance.
Adding a large number of edge-type-mismatch label functions produce variable performance that depends on the edge type being predicted and the label function's edge type source.

> Lastly, we show that this approach, even on this subgraph of Hetionet, could ascertain novel edges to Hetionet v1 with high confidence.

Reviewer: ```suggestion
Lastly, we show that this approach, even on this subgraph of Hetionet, could add new edges to Hetionet v1 with high confidence.
```

> We expect that future use in practice necessarily include additional filtering and scoring methods to further enhance precision.

Reviewer: ```suggestion
We expect that practical use of this strategy would include additional filtering and scoring methods which would further enhance precision.
```
